packages/create-rise/src/cli.ts

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env node
+import fs from 'node:fs'
+import path from 'node:path'
+
+import inquirer from 'inquirer'
+
+const cwd = process.cwd()
+
+const defaultProjectName = 'rise-project'
+
+async function prompt() {
+  let targetDir = ''
+  const answers = await inquirer.prompt([
+    {
+      type: 'input',
+      name: 'projectName',
+      message: 'Project Name',
+      default: defaultProjectName,
+    },
+    {
+      type: 'list',
+      when(ans) {
+        targetDir = formatTargetDir(ans.projectName)!
+        return fs.existsSync(targetDir) || !isEmpty(targetDir)
+      },
+      name: 'overwrite',
+      message: 'Target directory is not empty. Please choose how to proceed:',
+      choices: [
+        {
+          name: 'Remove existing files and continue',
+          value: 'yes',
+        },
+        {
+          name: 'Cancel operation',
+          value: 'no',
+        },
+        {
+          name: 'Ignore files and continue',
+          value: 'ignore',
+        },
+      ],
+    },
+  ])
+
+  const { projectName, overwrite } = answers
+  const root = path.join(cwd, projectName)
+
+  if (overwrite === 'yes') {
+    emptyDir(root)
+  } else if (!fs.existsSync(root)) {
+    fs.mkdirSync(root, { recursive: true })
+  }
+
+  const renameFiles: Record<string, string | undefined> = {
+    _gitignore: '.gitignore',
+  }
+
+  const template = 'template-react-navigation'
+  const templateDir = path.join(__dirname, `../${template}`)
+
+  const pkg = JSON.parse(fs.readFileSync(path.join(templateDir, `package.json`), 'utf-8'))
+
+  pkg.name = targetDir === '.' ? path.basename(path.resolve()) : targetDir
+
+  write('package.json', JSON.stringify(pkg, null, 2) + '\n')
+
+  const files = fs.readdirSync(templateDir)
+  for (const file of files.filter((f) => f !== 'package.json')) {
+    write(file)
```

Comment:
Question regarding performance: is it okay to leave this synchronous, or would it be better if `write` was async (and all `fs` functions were awaited), and e.g. files were copied with `Promise.all([write(), write()]`?

Reply:
The small number of files to copy should not cause any performance issues.

btw, I've copied the utils from `create-vite`